Automatic Validation ended with:
> 2024-01-27T06:33:23.1540889Z ##[error] Manifest Error: DisplayVersion declared in the manifest has overlap with existing DisplayVersion range in the index. Existing DisplayVersion range in index:  [ [VERSION, VERSION]]

(Automated response - build 669.)

--- a/manifests/c/CloudFoundry/CLI/v8/8.7.1/CloudFoundry.CLI.v8.installer.yaml
+++ b/manifests/c/CloudFoundry/CLI/v8/8.7.1/CloudFoundry.CLI.v8.installer.yaml
@@ -31,7 +31,7 @@ Installers:
   InstallerUrl: https://s3-us-west-1.amazonaws.com/v8-cf-cli-releases/releases/v8.7.1/cf8-cli-installer_8.7.1_winx64.zip
   InstallerSha256: 2B15E48A8B56E63B911ADEF36B5A54D401901B23B6636539BF1544B11E465DE2
   AppsAndFeaturesEntries:
-  - DisplayVersion: v8.7.2
+  - DisplayVersion: v8.7.1
     ProductCode: Cloud Foundry CLI_is1
   ElevationRequirement: elevatesSelf
 - Architecture: x86
@@ -42,7 +42,7 @@ Installers:
   InstallerUrl: https://s3-us-west-1.amazonaws.com/v8-cf-cli-releases/releases/v8.7.1/cf8-cli-installer_8.7.1_win32.zip
   InstallerSha256: D408F21753E2CB09D558DA832BC9D84F23DD1D081B4437D21BB42B3E8A782092
   AppsAndFeaturesEntries:
-  - DisplayVersion: v8.7.2
+  - DisplayVersion: v8.7.1
     ProductCode: Cloud Foundry CLI_is1
   ElevationRequirement: elevatesSelf
 ManifestType: installer

--- a/manifests/c/CloudFoundry/CLI/v8/8.7.1/CloudFoundry.CLI.v8.installer.yaml
+++ b/manifests/c/CloudFoundry/CLI/v8/8.7.1/CloudFoundry.CLI.v8.installer.yaml
@@ -31,7 +31,7 @@ Installers:
   InstallerUrl: https://s3-us-west-1.amazonaws.com/v8-cf-cli-releases/releases/v8.7.1/cf8-cli-installer_8.7.1_winx64.zip
   InstallerSha256: 2B15E48A8B56E63B911ADEF36B5A54D401901B23B6636539BF1544B11E465DE2
   AppsAndFeaturesEntries:
-  - DisplayVersion: VERSION
+  - DisplayVersion: v8.7.2
     ProductCode: Cloud Foundry CLI_is1
   ElevationRequirement: elevatesSelf
 - Architecture: x86
@@ -42,7 +42,7 @@ Installers:
   InstallerUrl: https://s3-us-west-1.amazonaws.com/v8-cf-cli-releases/releases/v8.7.1/cf8-cli-installer_8.7.1_win32.zip
   InstallerSha256: D408F21753E2CB09D558DA832BC9D84F23DD1D081B4437D21BB42B3E8A782092
   AppsAndFeaturesEntries:
-  - DisplayVersion: VERSION
+  - DisplayVersion: v8.7.2
     ProductCode: Cloud Foundry CLI_is1
   ElevationRequirement: elevatesSelf
 ManifestType: installer

--- a/manifests/c/CloudFoundry/CLI/v8/8.7.1/CloudFoundry.CLI.v8.installer.yaml
+++ b/manifests/c/CloudFoundry/CLI/v8/8.7.1/CloudFoundry.CLI.v8.installer.yaml
@@ -23,27 +23,5 @@ Installers:
     PortableCommandAlias: cf8
   InstallerUrl: https://s3-us-west-1.amazonaws.com/v8-cf-cli-releases/releases/v8.7.1/cf8-cli_8.7.1_win32.zip
   InstallerSha256: C4AD867E43A6F5E8F0C4FDEE77B1B7036E23BED455BCFF268E1811EA414E33AC
-- Architecture: x64
-  NestedInstallerType: inno
-  NestedInstallerFiles:
-  - RelativeFilePath: cf8_installer.exe
-  Scope: machine
-  InstallerUrl: https://s3-us-west-1.amazonaws.com/v8-cf-cli-releases/releases/v8.7.1/cf8-cli-installer_8.7.1_winx64.zip
-  InstallerSha256: 2B15E48A8B56E63B911ADEF36B5A54D401901B23B6636539BF1544B11E465DE2
-  AppsAndFeaturesEntries:
-  - DisplayVersion: v8.7.1
-    ProductCode: Cloud Foundry CLI_is1
-  ElevationRequirement: elevatesSelf
-- Architecture: x86
-  NestedInstallerType: inno
-  NestedInstallerFiles:
-  - RelativeFilePath: cf8_installer.exe
-  Scope: machine
-  InstallerUrl: https://s3-us-west-1.amazonaws.com/v8-cf-cli-releases/releases/v8.7.1/cf8-cli-installer_8.7.1_win32.zip
-  InstallerSha256: D408F21753E2CB09D558DA832BC9D84F23DD1D081B4437D21BB42B3E8A782092
-  AppsAndFeaturesEntries:
-  - DisplayVersion: v8.7.1
-    ProductCode: Cloud Foundry CLI_is1
-  ElevationRequirement: elevatesSelf
 ManifestType: installer
 ManifestVersion: 1.5.0

--- a/manifests/c/CloudFoundry/CLI/v8/8.7.1/CloudFoundry.CLI.v8.locale.en-US.yaml
+++ b/manifests/c/CloudFoundry/CLI/v8/8.7.1/CloudFoundry.CLI.v8.locale.en-US.yaml
@@ -9,7 +9,7 @@ PublisherUrl: https://cli.cloudfoundry.org/
 PublisherSupportUrl: https://github.com/cloudfoundry/cli/issues
 # PrivacyUrl: 
 # Author: 
-PackageName: Cloud Foundry CLI
+PackageName: Cloud Foundry CLI v8
 PackageUrl: https://cli.cloudfoundry.org/en-US/v8/
 License: Apache-2.0
 LicenseUrl: https://github.com/cloudfoundry/cli/blob/main/LICENSE

